Samples/kotlin/src/main/res/drawable/ic_expand_more_24px.xml

```diff
@@ -4,6 +4,6 @@
     android:viewportWidth="24"
     android:viewportHeight="24">
   <path
-      android:fillColor="#FF000000"
+      android:fillColor="#ff8e8e93"
```

Comment:
This shouldn't be black.  Should be this gray color according to design instead.